From d01d1a06e82a64cc0def8524c5ae6f9b057d765c Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Tue, 6 Jun 2023 22:41:21 +0200
Subject: [PATCH 1/2] Protobuf: Do not require version 3 do support Protobuf
 4.23.2 (23.2)

Signed-off-by: Silvio Traversaro <silvio@traversaro.it>
---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8d04ff2326..12da03b1d1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -176,9 +176,7 @@ set(IGN_UTILS_VER ${ignition-utils1_VERSION_MAJOR})
 
 #--------------------------------------
 # Find protobuf
-set(REQ_PROTOBUF_VER 3)
 ign_find_package(IgnProtobuf
-                 VERSION ${REQ_PROTOBUF_VER}
                  REQUIRED
                  COMPONENTS all
                  PRETTY Protobuf)

From b5060e35ee4b3af6b4860527dc778103731c149d Mon Sep 17 00:00:00 2001
From: Steve Peters <scpeters@openrobotics.org>
Date: Tue, 13 Jun 2023 11:51:01 -0700
Subject: [PATCH 2/2] TriggeredPublisher: don't catch FatalException

It has been removed from recent versions of protobuf.

Signed-off-by: Steve Peters <scpeters@openrobotics.org>
---
 src/systems/triggered_publisher/TriggeredPublisher.cc | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/src/systems/triggered_publisher/TriggeredPublisher.cc b/src/systems/triggered_publisher/TriggeredPublisher.cc
index 28f106584d..b8d1d70c0f 100644
--- a/src/systems/triggered_publisher/TriggeredPublisher.cc
+++ b/src/systems/triggered_publisher/TriggeredPublisher.cc
@@ -843,14 +843,7 @@ bool TriggeredPublisher::MatchInput(const transport::ProtoMsg &_inputMsg)
   return std::all_of(this->matchers.begin(), this->matchers.end(),
                      [&](const auto &_matcher)
                      {
-                       try
-                       {
-                         return _matcher->Match(_inputMsg);
-                       } catch (const google::protobuf::FatalException &err)
-                       {
-                          ignerr << err.what() << std::endl;
-                          return false;
-                       }
+                       return _matcher->Match(_inputMsg);
                      });
 }
 
