From 01baa51163b487830ebeebef2b44d487275e9333 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio.traversaro@iit.it>
Date: Sun, 28 Mar 2021 16:45:46 +0200
Subject: [PATCH] Update EntityComponentManager.hh

---
 include/ignition/gazebo/EntityComponentManager.hh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/ignition/gazebo/EntityComponentManager.hh b/include/ignition/gazebo/EntityComponentManager.hh
index 768432498..e2098d5fa 100644
--- a/include/ignition/gazebo/EntityComponentManager.hh
+++ b/include/ignition/gazebo/EntityComponentManager.hh
@@ -22,6 +22,7 @@
 
 #include <map>
 #include <memory>
+#include <optional>
 #include <set>
 #include <string>
 #include <typeinfo>
